cogs/modmail.py

```diff
@@ -448,7 +448,7 @@ async def send_scheduled_close_message(self, ctx, after, silent=False):
 
         embed = discord.Embed(
             title="Scheduled close",
-            description=f"This thread will close {silent}{human_delta}.",
+            description=f"This thread will close in {human_delta} {silent}.",
```

Comment:
There are some spacing issues here: always an extra space before the period and an extra space before "silently" when silently close.

I suggest changing the string to: `f"This thread will{' silently' if silent else ''} close in {human_delta}."` then remove [L447](https://github.com/modmail-dev/modmail/pull/3233/files#diff-bdc24e11b743f28e1f3139a9a78b2ee2c7bd0c71b132e396d57841e9998ccd90R447).

Reply:
I agree this format is better than both the current and proposed, makes it more polished for end user